### DISCUSSION TITLE
Show alternative text on images

### DISCUSSION THREAD

**Discussion by NickColley** [NickColley/semaphore#21] [New features and ideas]
Make it easier to see if a toot's image has alternative text.

**👉🏻  [View progress of this feature](https://github.com/NickColley/semaphore/pull/50)**

**Comment by NickColley** [comment]
Thinking that a simple and well established pattern is the `alt` badge where you click it and a dialog opens up.

<a href="https://help.twitter.com/en/using-twitter/how-to-use-alt-gif"><img width="230px" src="https://user-images.githubusercontent.com/2445413/213895872-aa28d196-5d6c-423e-ba11-68fa6fcf08fb.png" alt="How to use alt Twitter help page"/></a>

Though the word 'alt' is a technical word and not well understood, I can't think of a short alternative, if you were to use 'description' or 'image description' it would take up a lot of space?

Perhaps not having the badge over the image itself could allow for a clearer label for this feature without obscuring the image itself.

Welcome ideas on this.

**Comment by moan0s** [comment]
chaos.social currently tests a warning of posts with no alt-text which I really like. It is a red dotted frame around the media
![A screenshot of the mastodon interface with a red dotted frame around a chat gpt screenshot](https://user-images.githubusercontent.com/25753802/216850855-7a526377-4a4d-4350-8fb6-d19c0cd32e9a.png)
